### ISSUE TITLE
ACQ box with fid light in the box maybe should be orange warning?

### ISSUE THREAD

**Issue by jeanconn** [sot/starcheck#243]

**Comment by jeanconn** [comment]
This should be an orange warning (as the issue text indicates) and it also might make sense to adjust the acquisition probability for the box to 0 so that the overall acq probability doesn't count the acq star that would be very likely to be spoiled by the fid.  Then we'd get a failing acq probability in the summary if the spoiled star really mattered.

**Comment by jeanconn** [comment]
The warning for a fid light in a BOT box should continue to be a red warning (Critical) but could also have this probability adjustment if we go that way.

**Comment by taldcroft** [comment]
Agree fully with the first comment.  

I'm confused about the red warning for a fid light in a BOT box.  I assume that is a warning on the fid?  I don't see why being in a BOT/ACQ box matters for a fid, since fid search happens after acquisition and with a small (25") box.  The main issue is whether the star might spoil fid acquisition if the maneuver error is large enough.  That depends on the man err and relative mags.

**Comment by jeanconn** [comment]
Yes, the red warning is supposed to be for the FID.  I was under the impression that the small (25") box for the fid search is irrelevant, because if the the fid is in a BOT box, first it spoils the acquisition, but then it stays tracked in the box as the BOT star, because there is no new search on a BOT.  Fid search then fails because those pixels are already being tracked.  See "Failed fid acquisition in Obsid 14577".  I think this still might be a reason to stop using BOT stars (and always re-search).

**Comment by taldcroft** [comment]
Right, excellent point, thanks for reminding me.

Changing how we normally command star catalogs is a different and bigger question which would benefit from full SSAWG participation.  I think that warnings in starcheck largely mitigate the 14577 problem, and so we need to fully understand the other benefits / risks.

**Comment by jeanconn** [comment]
> Changing how we normally command star catalogs is a different and bigger question which would benefit from full SSAWG participation.

Indeed.  We started to talk about this at one point with regard to changing MAXMAG, but I didn't find that the data supported the idea that refining MAXMAG would give us a big improvement in tracking performance.  I just brought it up in this thread because it would fix this particular problem and it is worth keeping the idea of independently commanding ACQ and GUI in our box of tricks when looking at other issues.

**Comment by jeanconn** [comment]
Also, upon reflection I suppose "Yes, the red warning is supposed to be for the FID" is true but I suppose losing the guide star is worth the red warning too, so this issue would basically be worth 2 red warnings.
